.pre-commit-config.yaml

```diff
@@ -89,8 +89,8 @@ repos:
         stages: [commit, push]
 
 -   repo: https://github.com/pre-commit/mirrors-yapf
-    rev: v0.22.0
+    rev: v0.25.0
     hooks:
     -   id: yapf
-        args: ['-dpr']
+        args: ['-dipr']
```

Comment:
Have you tested this?
``` bash
usage: yapf [-h] [-v] [-d | -i] [-r | -l START-END] [-e PATTERN]
            [--style STYLE] [--style-help] [--no-local-style] [-p] [-vv]
            [files [files ...]]
yapf: error: argument -i/--in-place: not allowed with argument -d/--diff

```

Reply:
uhh, I totally missed that. Seems like we must run yapf for two passes if we want in-place formatting.